docs/Configuration.md

```diff
@@ -1074,15 +1074,15 @@ Setting this value to `fake` allows the use of fake timers for functions such as
 
 ### `transform` [object<string, string>]
 
-Default: `undefined`
+Default: `[{".*": "babel-jest"}]`
```

Comment:
Here's the default pattern:
https://github.com/facebook/jest/blob/be9ccf98a18f5d9edea872f5266fa41170d70a16/packages/jest-config/src/constants.ts#L11
The note should also recommend to use that one when explicitly configuring transform